WORKSPACE

```diff
@@ -319,6 +319,15 @@ http_file(
     urls = ["https://mirror.bazel.build/openjdk/azul-zulu11.37.48-ca-jdk11.0.6/zulu11.37.48-ca-jdk11.0.6-linux_aarch64-minimal-b23d4e05466f2aa1fdcd72d3d3a8e962206b64bf-1581690750.tar.gz"],
 )
 
+http_file(
+    name = "openjdk_linux_ppc64le_vanilla",
+    downloaded_file_path="adoptopenjdk-ppc64le-vanilla.tar.gz",
+    urls = [
```

Comment:
Please add the `sha256` attribute. I mirrored the file now and the SHA of the file is: `a417db0295b1f4b538ecbaf7c774f3a177fab9657a665940170936c0eca4e71a`.